ports/esp32/boards/DENEYAP_KART/board.json

```
@@ -0,0 +1,30 @@
+{
+    "deploy": [
+        "../deploy.md"
+    ],
+    "docs": "",
+    "features": [
+        "BLE",
+        "WiFi",
+        "SPIRAM",
+        "IMU",
+        "Microphone",
+        "Battery Charging",
+        "STEMMA QT/QWIIC",
+        "RGB LED",
+        "USB-MICRO",
+        "Breadboard friendly"
+    ],
+    "id": "dydk_mpv10",
+    "images": [
+        "deneyapkart.jpg"
+    ],
+    "mcu": "esp32",
+    "product": "Deneyap Kart",
+    "thumbnail": "",
+    "url": "https://deneyapkart.org/",
```

If possible, I believe that linking directly to the specific board is preferred.

```suggestion
    "url": "https://magaza.deneyapkart.org/en/product/detail/deneyap-kart",
```